docs/guides/admin/docs/modules/microsoftazuretranscripts.md

```diff
@@ -251,8 +251,25 @@ to the mediapackages and republishes it. Copy it into a new file under
 
 ```
 
+### Step 5: Install gstreamer
+
+The Microsoft speech-to-text client uses GStreamer to transcode audio stream.
+Therefore, GStreamer must be installed on your Opencast admin node.
+On Debian based systems, you can do that by running:
+
+```shell
+sudo apt-get install libgstreamer1.0-0 gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly
```

Comment:
This installs a whole bunch of things, some of which might be [problematic](https://gstreamer.freedesktop.org/modules/gst-plugins-ugly.html).  If these are hard requirements then we should be including a warning here with a link to the page above.  If they're *not* then I would suggest dropping `-ugly` at the very least.

Reply:
Here is what the [Speech SDK documentation](https://learn.microsoft.com/en-us/azure/cognitive-services/speech-service/how-to-use-codec-compressed-audio-input-streams?tabs=linux%2Ccentos%2Cjava-android%2Cterminal&pivots=programming-language-csharp) say about:

> The Speech SDK can use [GStreamer](https://gstreamer.freedesktop.org/) to handle compressed audio

In theory I've testet this module w/o gstreamer installed and it failed. Then I've started to install gstreamer-plugins-base, this also failed. This game was go on until all documented plugins are installed. So yes, the speech SDK needs at least gstreamer1-plugins-ugly-free (RedHat) or gstreamer1.0-plugins-ugly (Debian)!

Reply:
> > That's unfortunate. Is it [GStreamer] a dependency of `com.microsoft.cognitiveservices.speech:client-sdk`?
> 
> Do you know the answer to that question?

Yes, it is. This is documented [here](https://learn.microsoft.com/en-us/azure/cognitive-services/speech-service/how-to-use-codec-compressed-audio-input-streams?tabs=linux%2Ccentos%2Cjava-android%2Cterminal&pivots=programming-language-csharp).

Reply:
Fair enough.  I don't have to like it for it to be a hard dependency :D

Let's add it to the dos and be done with it.